.ci/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#Note: If add clearlinux as supported CI use a stateless os-release file
+source /etc/os-release
+
+if [ "$ID" == fedora ];then
+	sudo -E dnf -y install automake  bats
+elif [ "$ID" == ubuntu ];then
+	sudo add-apt-repository -y ppa:duggan/bats
```

Comment:
@jodh-intel  I need to add this repo because ubuntu from travis does not provide bats

Reply:
Ack. Could you add a comment explaining that bats isn't available for Ubuntu trusty then? That would make it easier to understand and if Travis ever offer Xenial, we then know we can drop this repo :)